examples/plugins/cmd-runner/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.8
+
+# Install kubectl
+# Note: Latest version may be found on:
+# https://aur.archlinux.org/packages/kubectl-bin/
+ADD https://storage.googleapis.com/kubernetes-release/release/v1.14.1/bin/linux/amd64/kubectl /usr/local/bin/kubectl
```

Comment:
Could steal this if you want a generic image: https://github.com/heptio/kube-conformance/blob/master/Makefile#L23-L25

Reply:
Yea I considered stealing that as well but I opted for the explicit version so people know what is in the image.